appveyor.yml

```diff
@@ -6,6 +6,7 @@ environment:
   global:
     # This will be used as part of the zipfile name
     PROJECT_NAME: rusoto
+    CARGO_INCREMENTAL: 1
```

Comment:
Wouldn't this _enable_ incremental compilation on AppVeyor? Or is this what you intended?